### PR TITLE
COR-44 Replace Keycloak with ORY Hydra

### PR DESCRIPTION
**Pull request by ludydoo**

### CODE REVIEW

File: api2/pkg/apps/cms/case_store.go [hunk 1]
```diff
@@ -71,15 +82,13 @@ func (s *Store) List(ctx context.Context, listOptions ListOptions) (*CaseList, e
...
-func (s *Store) Update(ctx context.Context, kase *Case) error {
+func (s *CaseStore) Update(ctx context.Context, kase *Case) error {
 	_, err := s.collection.UpdateOne(ctx, bson.M{
 		"id": kase.ID,
 	}, bson.M{
 		"$set": bson.M{
 			"description": kase.Description,
 			"done":        kase.Done,
-			"parentId":    kase.ParentID,
-			"teamId":      kase.TeamID,
```
Comment (nilueps) [inline api2/pkg/apps/cms/case_store.go:82]: Why were these two lines removed?

Comment (ludydoo) [reply api2/pkg/apps/cms/case_store.go:82]: I figured we wanted to make these readonly (not able to be updated)

File: api2/pkg/apps/cms/init.go [hunk 2]
```diff
@@ -0,0 +1 @@
+package cms
```
Comment (nilueps) [inline api2/pkg/apps/cms/init.go:1]: unecessary file?

Comment (ludydoo) [reply api2/pkg/apps/cms/init.go:1]: indeed ;)

File: api2/pkg/apps/iam/client.go [hunk 3]
```diff
@@ -0,0 +1,75 @@
+package iam
+
+import "github.com/nrc-no/core-kafka/pkg/rest"
+
+type ClientSet struct {
+	c *rest.Client
```
Comment (nilueps) [inline api2/pkg/apps/iam/client.go:6]: consider renaming
```suggestion
	restClient *rest.Client
```

File: api2/pkg/apps/iam/organization_list.go [hunk 4]
```diff
@@ -0,0 +1,7 @@
+package iam
+
+import "net/http"
+
+func (s *Server) ListOrganizations(w http.ResponseWriter, req *http.Request) {
+
```
Comment (nilueps) [inline api2/pkg/apps/iam/organization_list.go:6]:
```suggestion
    // TODO
```

File: api2/pkg/apps/iam/organization_post.go [hunk 5]
```diff
@@ -0,0 +1,7 @@
+package iam
+
+import "net/http"
+
+func (s *Server) PostOrganization(w http.ResponseWriter, req *http.Request) {
+
```
Comment (nilueps) [inline api2/pkg/apps/iam/organization_post.go:6]:
```suggestion
    // TODO
```

File: api2/pkg/apps/iam/organization_put.go [hunk 6]
```diff
@@ -0,0 +1,7 @@
+package iam
+
+import "net/http"
+
+func (s *Server) PutOrganization(w http.ResponseWriter, req *http.Request) {
+
```
Comment (nilueps) [inline api2/pkg/apps/iam/organization_put.go:6]:
```suggestion
    // TODO
```

File: api2/pkg/apps/iam/static.go [hunk 7]
```diff
@@ -0,0 +1,230 @@
...
+		{
+			PartyTypeRule: &PartyTypeRule{
+				FirstPartyTypeID:  IndividualPartyType.ID,
+				SecondPartyTypeID: IndividualPartyType.ID,
+			},
+		},
+	},
+}
+
+var FilialRelationshipType = RelationshipType{
```
Comment (nilueps) [inline api2/pkg/apps/iam/static.go:169]:
```suggestion
var SiblingRelationshipType = RelationshipType{
```
'Filial' suggest a "son of" or "daughter of" relationship.

**Review by nilueps — CHANGES_REQUESTED** [review]
Can't run it on my end. Tried deleting local branch, re-cloning, repo.

**Review by nilueps — APPROVED** [review]

**Pull request merged**